devel/git-tools/Portfile

```
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            MestreLion git-tools 40f8174bf0f4db100c902be7e883dd70cfd1995f
+version                 20210806
+
+checksums               rmd160  17a570b3644306c3bf1f7fcea7c8d06502f52b70 \
+                        sha256  859504bc65a447352c99d0595a92214d24bd8dd97ebb7e6b3ec3b0c19742dc56 \
+                        size    32998
+
+maintainers             {@telotortium gmail.com:rirelan} openmaintainer
+platforms               darwin
+categories              devel
+supported_archs         noarch
+
+description             Assorted git-related scripts and tools, including git-restore-mtime
+long_description        ${description} -- \
+                        Included tools: git-branches-rename, git-clone-subset, \
+                        git-find-uncommitted-repos, git-rebase-theirs, \
+                        git-restore-mtime, git-strip-merge
+license                 GPL-3
+
+use_configure           no
+
+build {}
+destroot {
+    set mandir ${destroot}${prefix}/share/man/man1
+    set bindir ${destroot}${prefix}/bin
+    xinstall -d ${mandir}
+    xinstall -m 644 {*}[glob ${worksrcpath}/man1/*.1] ${mandir}
+    xinstall -d ${bindir}
+    xinstall -m 755 {*}[glob ${worksrcpath}/git-*] ${bindir}
```

MacPorts creates standard directories for you.
```suggestion
    xinstall -m 0444 {*}[glob ${worksrcpath}/man1/*.1] ${destroot}${prefix}/share/man/man1
    xinstall -m 0755 {*}[glob ${worksrcpath}/git-*] ${destroot}${prefix}/bin
```